src/index.js

```diff
@@ -0,0 +1,115 @@
+const tasks = [
+  {
+    description: 'Go grocery shopping',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Finish homework',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'Take out the trash',
+    completed: false,
+    index: 3,
+  },
+];
+
+function createListItem(task) {
+  const listItem = document.createElement('li');
+  listItem.innerHTML = `
+    <div class="task-item">
+      <div class="task-item__checkbox">
+        <input type="checkbox" id="task-${task.index}" ${task.completed ? 'checked' : ''}>
+        <label for="task-${task.index}"></label>
+      </div>
+      <div class="task-item__description ${task.completed ? 'completed' : ''}">
+        ${task.description}
+      </div>
+      <div class="task-item__delete">
+        <button class="delete-button" data-index="${task.index}">
+          <i class="fas fa-trash-alt"></i>
+        </button>
+      </div>
+    </div>
```

Comment:
- [ ] Once you have fixed all the above changes and make sure that your application is running, you should ensure that created CSS is  used to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) as mentioned in the project requirements. Right now it is difficult to check as you didn't correctly setup your Webpack dev server